FieldOpt/CMakeLists.txt

```diff
@@ -24,7 +24,8 @@ set(Boost_USE_MULTITHREAD ON) # enable multithreading
 find_package(Boost COMPONENTS REQUIRED program_options)
 
 find_package(Threads)
-find_package(Eigen3)
+# find_package(Eigen3)
+set( EIGEN3_INCLUDE_DIR "$ENV{EIGEN3_INCLUDE_DIR}" )
```

Comment:
Just a thing that has to do with the arch distro on my laptop, sont know why 8t complains. Remove the commit (I forgot when making the pull request). M.

Sent from my Sony Xperia™ smartphone


---- Einar Baumann wrote ----


In FieldOpt/CMakeLists.txt<https://github.com/PetroleumCyberneticsGroup/FieldOpt/pull/43#discussion_r72035984>:

> @@ -24,7 +24,8 @@ set(Boost_USE_MULTITHREAD ON) # enable multithreading
>  find_package(Boost COMPONENTS REQUIRED program_options)
>
>  find_package(Threads)
> -find_package(Eigen3)
> +# find_package(Eigen3)
> +set( EIGEN3_INCLUDE_DIR "$ENV{EIGEN3_INCLUDE_DIR}" )


Was this neccessary? It shouldn't have been. find_package uses the EIGEN3_INCLUDE_DIR variable to find the location.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/PetroleumCyberneticsGroup/FieldOpt/pull/43/files/ade08f8ba8203ecf825c5f2b8762c37833be1030#r72035984>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHAlY6I9Dy3t8TPoPpwIyOXhOhUYo1ENks5qZIXYgaJpZM4JTeC5>.